### PR TITLE
chore: simplify renovate bot

### PR DESCRIPTION
**Pull request by Boshen**
This is for keeping the notification noises down.

I'll update my config https://github.com/Boshen/renovate when I see things is not getting updated.

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -1,58 +1,28 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:best-practices",
```
Comment (Boshen) [inline .github/renovate.json:4]: We don't need anything from it https://docs.renovatebot.com/presets-config/#configrecommended

File: .github/renovate.json [hunk 2]
```diff
@@ -1,58 +1,28 @@
...
-    ":automergeRequireAllStatusChecks",
-    ":automergePr"
-  ],
-  "assignees": ["hyf0", "IWANABETHATGUY", "underfin"],
-  "assigneesSampleSize": 1,
-  "timezone": "Asia/Shanghai",
-  "schedule": ["after 1pm", "before 4pm"],
```
Comment (Boshen) [inline .github/renovate.json:11]: To reduce the noise, my config is set to once a week, "before 10am on monday".

File: .github/renovate.json [hunk 3]
```diff
@@ -1,58 +1,28 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:best-practices",
-    ":automergeRequireAllStatusChecks",
-    ":automergePr"
-  ],
-  "assignees": ["hyf0", "IWANABETHATGUY", "underfin"],
```
Comment (Boshen) [inline .github/renovate.json:8]: We don't need them assigned, PR will show up if they fail to upgrade.

**Review by hyf0 — DISMISSED** [review]

**Comment by Boshen** [comment]
I'm going to merge this and trigger the bot and see what happens.

**Pull request merged**